app/views/screens/_form.html.erb

```diff
@@ -132,13 +132,36 @@
                 <label class="block text-sm font-medium text-gray-700 mb-2">
                   <%= field.name.titleize %> Field
                 </label>
+
+                <!-- Pinned Content -->
                 <%= fc.collection_select :pinned_content_id,
                     @active_content,
                     :id,
                     :name,
                     { include_blank: "No pinned content" },
                     { class: "block w-full px-3 py-2 border border-gray-300 rounded-lg shadow-sm text-sm focus:outline-none focus:ring-2 focus:ring-blue-500 focus:border-blue-500" } %>
                 <p class="text-xs text-gray-500 mt-1">Select content to pin, or "No pinned content" to clear</p>
+
+                <!-- Content Ordering Strategy -->
+                <div class="mt-3">
+                  <%= fc.label :ordering_strategy, "Content Ordering", class: "block text-sm font-medium text-gray-700 mb-2" %>
+                  <%= fc.select :ordering_strategy,
+                      options_for_select(
+                        [
+                          ["Random (Default)", "random"],
```

Comment:
Is it worth it to somehow use the `STRATEGIES` from `ContentOrderers` instead of duplicating these strings?